### DISCUSSION TITLE
The Admin Tool can only be executed in the pod, and I think it would be more reasonable to integrate relevant functions into the proxy

### DISCUSSION THREAD

**Discussion by charlienss** [apache/rocketmq#6620] [Q&A]
Can it connect to proxy?I need performance pressure test data in this area.

**Comment by drpmma** [comment]
Related discussion https://github.com/apache/rocketmq/issues/6603

There's no plan to support admin interface in proxy for now. Please use nameserver as access endpoint instead.
